dxr/plugins/clang/condense.py

```diff
@@ -315,10 +315,9 @@ def listify_keys(d):
     condense(
         lines_from_csvs(csv_folder, '*.csv'),
         {'impl': partial(process_impl, parents, children),
-         'function': partial(process_override, overrides, overriddens)},
-        predicate=lambda kind, fields: (kind == 'function' and
-                                        'overriddenname' in fields) or
-                                       kind == 'impl')
+         'func_override': partial(process_override, overrides, overriddens)},
```

Comment:
Kudos on a very good design decision. It's so much better to have a dedicated kind for overrides rather than trying to shoehorn a multi-valued key into `function`.